src/utils/array.cairo

```diff
@@ -110,3 +110,8 @@ func contains{range_check_ptr}(arr_len: felt, arr: felt*, value: felt) -> felt {
     }
     return FALSE;
 }
+
+func pad_end(arr: felt*, value: felt, n: felt) {
```

Comment:
add a check on the actual vs target length, see js syntax https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/padEnd

```
padEnd(targetLength)
```